communication.md

```diff
@@ -1,84 +1 @@
-Discussions on most channels are archived at [kubernetes.slackarchive.io].
-Start archiving by inviting the _slackarchive_ bot to a
-channel via `/invite @slackarchive`.
-To add new channels, contact one of the admins in the #slack-admins channel. Our guidelines are [here](/communication/slack-guidelines.md).
-
-## Issues
-
-If you have a question about Kubernetes or have a problem using it,
-please start with the [troubleshooting guide].
-
-If that doesn't answer your questions, or if you think you found a bug,
-please [file an issue].
-
-
-## Mailing lists
-
-Development announcements and discussions appear on the Google group
-[kubernetes-dev] (send mail to `kubernetes-dev@googlegroups.com`).
-
-Users trade notes on the Google group
-[kubernetes-users] (send mail to `kubernetes-users@googlegroups.com`).
-
-## Office Hours
-
-Office hours are held once a month. Please refer to [this document](events/office-hours.md) to learn more.
-
-## Weekly Meeting
-
-Refer to the [community meeting document](events/community-meeting.md) for information on our weekly meeting. 
-
-## Conferences
-
-Kubernetes is the main focus of CloudNativeCon/KubeCon, held every spring in Europe and winter in North America. Information about these and other community events is available on the CNCF [events] pages.
-
-
-[Blog]: http://blog.kubernetes.io
-[calendar.google.com]: https://calendar.google.com/calendar/embed?src=cgnt364vd8s86hr2phapfjc6uk%40group.calendar.google.com&ctz=America/Los_Angeles
-[CNCF code of conduct]: https://github.com/cncf/foundation/blob/master/code-of-conduct.md
-[communication]: /communication.md
-[community meeting]: /communication.md#weekly-meeting
-[events]: https://www.cncf.io/events/
-[file an issue]: https://github.com/kubernetes/kubernetes/issues/new
-[Google+]: https://plus.google.com/u/0/b/116512812300813784482/116512812300813784482
-[iCal url]: https://calendar.google.com/calendar/ical/cgnt364vd8s86hr2phapfjc6uk%40group.calendar.google.com/public/basic.ics
-[Kubernetes Community Meeting Agenda]: https://docs.google.com/document/d/1VQDIAB0OqiSjIHI8AWMvSdceWhnz56jNpZrLs6o7NJY/edit#
-[kubernetes-community-video-chat]: https://groups.google.com/forum/#!forum/kubernetes-community-video-chat
-[kubernetes-dev]: https://groups.google.com/forum/#!forum/kubernetes-dev
-[kubernetes-users]: https://groups.google.com/forum/#!forum/kubernetes-users
-[kubernetes.slackarchive.io]: https://kubernetes.slackarchive.io
-[kubernetes.slack.com]: https://kubernetes.slack.com
-[Slack]: http://slack.k8s.io
-[Special Interest Group]: /README.md#SIGs
-[Stack Overflow]: http://stackoverflow.com/questions/tagged/kubernetes
-[timezone table]: https://www.google.com/search?q=1000+am+in+pst
-[troubleshooting guide]: http://kubernetes.io/docs/troubleshooting
-[Twitter]: https://twitter.com/kubernetesio
-[weekly meeting]: https://zoom.us/my/kubernetescommunity
+**NOTE:** This document has moved to [a new location](/community/communication].
```

Comment:
Could we put a tombstone end date here, similar to what @spiffxp did here: https://github.com/kubernetes/community/blob/master/contributors/devel/security-release-process.md